runtime/static_config.go

```diff
@@ -267,8 +265,34 @@ func (conf *StaticConfig) MustGetStruct(key string, ptr interface{}) {
 	panic(errors.Errorf("Key (%s) not available", key))
 }
 
+// SetConfigValue sets the static config value.
+// dataType can be a boolean, number or string.
+// SetConfigValue will panic if the config is frozen.
+func (conf *StaticConfig) SetConfigValue(key string, bytes []byte, dataType string) {
```

Comment:
This isn't clearly named because it doesn't distinguish itself clearly from `SetOrDie`. Might be better named as `SetRawOrDie`